packages/flutter/test/material/paginated_data_table_test.dart

```diff
@@ -786,4 +786,34 @@ void main() {
     await tester.pumpWidget(buildTable(false));
     expect(find.byType(Checkbox), findsNothing);
   });
+
+  testWidgets('Table should not use decoration from DataTableTheme', (WidgetTester tester) async {
+    final Size originalSize = binding.renderView.size;
+    await binding.setSurfaceSize(const Size(800, 800));
+
+    Widget buildTable() => MaterialApp(
```

Comment:
Prefer curly braces over `=>` for everything except one-liners that return a value.

Reply:
Done